modules/ROOT/pages/sid-reference.adoc

```diff
@@ -0,0 +1,244 @@
+= Sensitive Information Detection
+
+Sensitive Information Detection (SID) allows you to identify where you have potential leakage of sensitive information.
+
+These are some common scenarios where implementing SID can help you keep sensitive data more secure: 
+
+* Developers use sensitive information when they build apps. If developers aren’t aware of what is sensitive information and what is not, they can inadvertently cause leakage by exposing the sensitive information.
+* Developers build test APIs, or add testing parameters to existing APIs, that return sensitive information. These APIs or parameters were not documented, and the developer forgets to remove the sensitive information from the API.  
+* Data is received from partners that may contain sensitive information, which should be cleansed.
+
+
+== WAF Security Policy Summary Log
+
+When you apply the WAF policy to the Runtime Fabric, a `rtfWafSecurityPolicySummary` log is generated, which shows the WAF category, rule ID, and count for how many times the violation was detected during the last collection interval. 
+For example:
+
+[json]
+----
+{
+	"timeStamp": "2019-04-15T16:13:00.045Z",
+	"collectionIntervalInMinutes": "1",
+	"node": "node-host",
+	"wafSummary": {
+		"requestScannerDetection": [
+
+		],
+		"requestProtocolEnforcement": [
+
+		],
+		"requestProtocolAttack": [
+
+		],
+		"requestApplicationAttackLfi": [
+
+		],
+		"requestApplicationAttackRfi": [
+
+		],
+		"requestApplicationAttackRce": [
+
+		],
+		"requestApplicationAttackPhp": [
+
+		],
+		"requestApplicationAttackXss": [
+
+		],
+		"requestApplicationAttackSqli": [
+
+		],
+		"requestApplicationAttackSessionFixation": [
+
+		],
+		"requestSidPiiCore": [{
+			"ruleId": "600100",
+			"count": 1
+		}],
+		"requestSidPiiEnhanced": [
+
+		],
+		"requestSidPci": [
+
+		],
+		"requestSidPhi": [
+
+		],
+		"responseDataLeakages": [
+
+		],
+		"responseDataLeakagesSql": [
+
+		],
+		"responseDataLeakagesJava": [
+
+		],
+		"responseDataLeakagesPhp": [
+
+		],
+		"responseDataLeakagesIis": [
+
+		],
+		"responseSidPiiCore": [
+
+		],
+		"responseSidPiiEnhanced": [
+
+		],
+		"responseSidPci": [
+
+		],
+		"responseSidPhi": [
+
+		]
+	}
+}
+----
+
+== SID Summary Log
+
+When you apply the WAF policy to the Runtime Fabric, a `rtfSidSummary` log is generated, which shows the offending API, rule description, direction, and count for the number of times the SID rule was triggered. 
+For example:
+
+[json]
+----
+{
+	"timeStamp": "2019-04-15T16:13:00.045Z",
+	"collectionIntervalInMinutes": "1",
+	"node": "node-host-name",
+	"sidSummary": {
+		"api": "/Post",
+		"rule": "TAXID",
+		"direction": "Request",
+		"count": 1
+	}
+}
+----
+
+== Rule Categories and IDs
+
+This table lists the rule categories and rule IDs for SID that are returned in the logs.
```

Comment:
I don't think we should abbreviate SID (or we should abbreviate it everywhere in a topic after first occurrence), but regardless, data is a mass noun so in American English, it should be "...for SID that is returned in the logs."